python/rascaline/rascaline/utils/_classes.py

```diff
@@ -0,0 +1,20 @@
+from equistore.core import Labels, TensorBlock, TensorMap
+
+from ..calculator_base import CalculatorBase
+from ..systems import IntoSystem
+from . import _dispatch
```

Comment:
Do you want to change this before merging?

Reply:
instead of making `_dispatch` a part of `_classes`, just import it `from . import _dispatch` in the code, like we do for the operations